src/targets/java/okhttp.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @description
+ * HTTP code snippet generator for Java using OkHttp.
+ *
+ * @author
+ * @shashiranjan84
+ *
+ * for any questions or issues regarding the generated code snippet, please open an issue mentioning the author.
+ */
+
+'use strict'
+
+var util = require('util')
+var CodeBuilder = require('../../helpers/code-builder')
+
+module.exports = function (source, options) {
+  var opts = util._extend({
+    indent: '  '
+  }, options)
+
+  var code = new CodeBuilder(opts.indent)
+
+  var methods = [ 'GET', 'POST', 'PUT', 'DELETE', 'PATCH', 'HEAD']
+
+  var methodsWithBody = [ 'POST', 'PUT', 'DELETE', 'PATCH']
```

Comment:
isn't this redundant? all methods could technically have bodies! should just focus on whether the postData has content instrlead.